rust/arrow/src/compute/kernels/aggregate.rs

```diff
@@ -149,4 +202,18 @@ mod tests {
         assert_eq!(5, min(&a).unwrap());
         assert_eq!(9, max(&a).unwrap());
     }
+
+    #[test]
+    fn test_string_min_max_with_nulls() {
+        let a = StringArray::from(vec![Some("b"), None, None, Some("a"), Some("c")]);
+        assert_eq!("a", min_string(&a).unwrap());
+        assert_eq!("c", max_string(&a).unwrap());
+    }
+
+    #[test]
+    fn test_string_min_max_all_nulls() {
```

Comment:
Could you add an additional test with an array with one or more `None` at the start, followed by one or more `Some` to see if I am correct about my earlier comment?

Reply:
done